test/e2e/user-actions-benchmark.ts

```diff
@@ -24,8 +24,10 @@ async function loadNewAccount(): Promise<number> {
   await withFixtures(
     {
       fixtures: new FixtureBuilder().build(),
-      localNodeOptions: 'ganache',
       disableServerMochaToBackground: true,
+      localNodeOptions: {
+        accounts: 1,
```

Comment:
setting custom account 1, as in this test, we want that the second account we create has 0 balance.
Before, it was already like this as in ganache we only seeded the first account, as we were using the accounts option instead of the SRP (which seeds 10 accounts by default)